src/redux/books/books.js

```diff
@@ -1,35 +1,38 @@
-const ADD_BOOK = 'bookstore/books/ADD_BOOKS';
-const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOKS';
+/* eslint-disable prefer-template */
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const ADD_ARR_BOOK = 'bookstore/books/ADD_ARR_BOOK';
 
-const initialState = [{
-  id: '0', title: 'Viaje al centro de la tierra', author: 'Erik', category: 'action',
-},
-{
-  id: '1', title: 'How to programm in java 7', author: 'Federico', category: 'Romance',
-}];
+const initialState = [];
 
 const reducer = (state = initialState, action) => {
   switch (action.type) {
-    case ADD_BOOK:
+    case ADD_BOOK + '/fulfilled':
       return [
         ...state,
-        {
-          id: action.id,
-          title: action.title,
-          author: action.author,
-          category: action.category,
-        },
+        action.payload,
       ];
 
-    case REMOVE_BOOK:
-      return state.filter((item) => item.id !== action.id);
+    case REMOVE_BOOK + '/fulfilled':
+      return state.filter((item) => item.id !== action.payload);
+
+    case ADD_ARR_BOOK + '/fulfilled':
+      return [
+        ...action.payload,
+      ];
 
     default:
       return state;
   }
 };
 
 // action creators
+// this one was before use the createAsyncTunk
+// const addArrBook = (arr) => ({
+//   type: ADD_ARR_BOOK,
+//   arrBook: arr,
+// });
```

Comment:
- [ ] Kindly make sure that you remove all commented-out/ unwanted pieces of code from your code base. It makes it look messy and cluttered.